client/src/commands/closeSession.ts

```diff
@@ -15,4 +14,5 @@ export async function closeSession(message?: string): Promise<void> {
   if (message) {
     window.showInformationMessage(message);
   }
+  commands.executeCommand("setContext", "SAS.authorized", false);
```

Comment:
Now that the AuthenticationProvider has been merged to master. It would be great for the content navigator to NOT rely on the compute session. It doesn't really require setting up a compute session, and won't be impacted by compute session close.
It can get the access token from authentication provider. When needed the AuthenticationProviderr can set UI context as appropriate.

Reply:
OK, sounds like you don't have access right to your user name... I didn't hear this kind of thing before. If that's a valid case, we probably should try catch this part.

Reply:
Cool. Thank you for catching and fixing it!

Reply:
Hey, but, this line now can be removed right? Here user doesn't sign out but just close a compute session.

Reply:
@scottdover, would you check this?

Reply:
Hey @scnwwu . It appears logging in is currently not working as expected, and I'm unable to verify this change. Unless there's any further things to resolve, I'd recommend getting this one merged in and adjusting this command as needed after login issues are resolved

Reply:
I know the server now has a CSP issue I'll have to communicate it with the team separately. It's not currently anything can be done in client code.
Here's a workaround for now. Click "Copy" for the URL instead of "Open", and paste it into Firefox. Firefox doesn't have such issue.
However, your last screenshot in #131 seems another problem. It seems you have to check the checkbox before Authorize Access.
If you still have issues to authorize, I'm OK to check it after merge. Thanks.

Reply:
One thing to call out is that you _can't_ copy if your endpoint is configured as a trusted domain. Once I removed it from the list and removed that line, things worked as expected.